script-base.js

```diff
@@ -268,7 +315,7 @@ Generator.prototype.addGradlePlugin = function (group, name, version) {
             file: fullPath,
             needle: 'jhipster-needle-gradle-buildscript-dependency',
             splicable: [
-                'classpath group: \'' + group + '\', name: \'' + name + '\', version: \'' + version + '\''
+                    'classpath group: \'' + group + ', name: \'' + name + ', version: \'' + version + '\''
```

Comment:
can you review this ? see https://github.com/jhipster/generator-jhipster/pull/2479